packages/teleport-shared/src/utils/uidl-utils.ts

```diff
@@ -424,3 +424,16 @@ export const transformAttributesAssignmentsToJson = (
 
   return newStyleObject
 }
+
+export const traverseUIDLForRootElmNode = (node) => {
```

Comment:
I would name this: `findFirstElementNode` because the traverse functions don't return something necessarily, they call a function on the node. in this case you are returning a specific node.

Reply:
also, add the typing for the `node`